cmd/crc/cmd/start.go

```diff
@@ -63,8 +63,11 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 
 	checkIfNewVersionAvailable(config.Get(cmdConfig.DisableUpdateCheck).AsBool())
 
-	if err := preflight.StartPreflightChecks(config); err != nil {
-		return nil, err
+	// If skip-all-preflight-checks is true then don't run the prefligh checks
+	if !config.Get(cmdConfig.SkipAllPreflightChecks).AsBool() {
```

Comment:
add log that all preflights were explicitly skipped

Reply:
Added info log as..  `All preflight checks are skipped`